.devops/templates/build-test-lint.yml

```diff
@@ -7,7 +7,7 @@ steps:
     displayName: yarn (install packages)
 
   - script: |
-      yarn nx run @fluentui/workspace-plugin:check-graph
+      yarn nx run workspace-plugin:check-graph
       yarn nx g @fluentui/workspace-plugin:tsconfig-base-all --verify
```

Comment:
note: nx plugin collection needs to always refer to npmPackageName - thus scope is still used